.github/workflows/docker-multiarch.yml

```diff
@@ -0,0 +1,64 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Can we keep the branch name as something unique (call it 'only-demo' or something) so it doesn't get triggered in github CI, but we use the logic on a builder in our infra to push it to DOCKER? With that, this is good to get in to repo.

Reply:
We can keep the docker actions as a reference. Post-merge, these actions (multiarch, docker) can be disabled at the repository level.